build/verify-features.ts

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env -S TS_NODE_COMPILER_OPTIONS='{"module":"es2020"}' node --loader ts-node/esm
+
+import {readdirSync} from 'node:fs';
+
+import {getFeaturesMeta, getImportedFeatures} from './readme-parser.js'; // Must import as `.js`
+
+const featuresDirContents = readdirSync('source/features/');
+const importedFeatures = getImportedFeatures();
+const featuresInReadme = getFeaturesMeta();
+
+const errors: string[] = [];
+
+for (let fileName of featuresDirContents) {
+	if (fileName === 'index.tsx' || fileName.endsWith('.css') || fileName.startsWith('rgh-')) {
+		continue;
+	}
+
+	if (!fileName.endsWith('.tsx')) {
+		errors.push(`fileext: The \`/source/features\` folder should only contain .css and .tsx files. File \`${fileName}\` violates that rule.`);
+		continue;
+	}
+
+	fileName = fileName.replace('.tsx', '');
+
+	const featureMeta = featuresInReadme.find(feature => feature.id === fileName);
+	if (!featureMeta) {
+		errors.push(`readme: The feature ${fileName} is not included in the readme.`);
+		continue;
+	}
+
+	if (featureMeta.description.length < 20) {
+		errors.push(`desc: The description for ${featureMeta.id} is less than 20 characters. Try explaining it better!`);
+	}
+
+	if (!importedFeatures.includes(featureMeta.id)) {
```

Comment:
Let's move this check to the second place and then move this check after:

```js
 	if (fileName.startsWith('rgh-')) {
 		continue;
 	}
```

This way `rgh` features will still be covered by the first 2 rules

Reply:
The RGH features are not supposed to be described in the README right?

Reply:
I didn't understand which checks the `rgh-` files should be covered by

Reply:
Done in my last commit, I hope it works.

![borat-picture-a287a51073c07cbc5566aff98c64beb780bde49b](https://user-images.githubusercontent.com/1402241/130597948-fa750a98-c0e8-4188-8477-181f6399c474.jpg)

Reply:
It works, thanks